src/xavier/gui.py

```diff
@@ -283,9 +283,7 @@ def launch_gui(DEBUG=True):
                 genome=genome,
                 targets=values["-TARGETS-"]
                 if values["-TARGETS-"]
```

Comment:
We can simplify this line since "-TARGETS-" is guaranteed to either be a string or an empty string.

```suggestion
                targets=values["-TARGETS-"],
```